packages/flutter/lib/src/widgets/image.dart

```diff
@@ -207,6 +207,31 @@ typedef ImageLoadingBuilder = Widget Function(
 /// The image is painted using [paintImage], which describes the meanings of the
 /// various fields on this class in more detail.
 ///
+/// {@tool sample}
+/// The default constructor can be used with any [ImageProvider], such as a
+/// [NetworkImage], to display an image from the internet.
+///
+/// ![An image of an owl displayed by the image widget](https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg)
+///
+/// ```dart
+/// const Image(
+///   image: NetworkImage('https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg'),
+/// )
+/// ```
+/// {@end-tool}
+///
+/// {@tool sample}
+/// The [Image] Widget also provides several constructors to display different
+/// types of images for convenience. In this example, use the [Image.network]
+/// constructor to display an image from the internet.
+///
+/// ![An image of an owl displayed by the image widget using the shortcut constructor](https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg)
```

Comment:
Maybe we can add a different image here? Instead of the same image twice?

Reply:
Yes, we can certainly do that! I grabbed the `owl.jpg` from the original Flutter website repository and copied it to the assets-for-api-docs repo since I figured the copyright to use that image was taken care of. However, I didn't find any more images like that one in the website repo nor in the assets-for-api-docs repo.

Just to ensure I'm not running afoul of copyright laws, do you know of any approved images we could use for this second example or where I might be able to find one that works?

Reply:
Great point! I'm adding another image of an owl that is ok to use. (See https://github.com/flutter/assets-for-api-docs/pull/95). Once that lands, feel free to use that image here. :)

Reply:
That image is now available. 👍

Reply:
Thanks so much! I've swapped the second example to use the new image.